qiita_core/support_files/server.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Can you post a screenshot of the error rather than copy paste? I would like to see better the error from chrome.

Thanks,